app/models/settings.rb

```diff
@@ -0,0 +1,40 @@
+# == Schema Information
+#
+# Table name: settings
+#
+#  id                  :integer          not null, primary key
+#  created_at          :datetime
+#  updated_at          :datetime
+#  singleton_guard     :integer
+#  current_semester_id :integer
+#  npo_app_open        :boolean
+#  student_app_open    :boolean
+#
+
+class Settings < ActiveRecord::Base
+  validates_numericality_of :singleton_guard, equal: 0
+  validates :npo_app_open, inclusion: { in: [true, false] }
+  validates :student_app_open, inclusion: { in: [true, false] }
+  validates :current_semester_id, presence: true
+
+  # source: http://stackoverflow.com/questions/399447/how-to-implement-a-singleton-model
+  def self.instance
+    # there will be only one row, and its ID must be '1'
+    begin
+      find(1)
```

Comment:
I'm not sure this is a super good idea cause in the case that someone the first Settings object gets deleted our app will try to create a new one but the way the DB works is that no other settings object will ever have the id = 1 unless you set it manually (which we don't wanna do).

Maybe we can just do

    def self.instance
      first || create(npo_app_open: false, student_app_open: false,
                      current_semester_id: nil, singleton_guard: 0)
    end